File: app/src/main/java/com/neome/java/api/core/user/msg/MsgSignIn.java
// neome.ai API - do not change
//

package com.neome.java.api.core.user.msg;

import com.neome.java.api.core.base.msg.MsgHandle;
import com.neome.java.api.meta.base.Types.EnumDeviceType;

@SuppressWarnings("unused")
public class MsgSignIn extends MsgHandle
{
  public String deviceName;

  public EnumDeviceType deviceType;

  public String password;

  public boolean rememberMe;
}

File: app/src/main/java/com/neome/api/core/user/msg/MsgSignIn.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.user.msg

import com.neome.api.meta.base.Types.EnumDeviceType
import com.neome.api.core.base.msg.MsgHandle

interface MsgSignIn : MsgHandle
{
  val deviceName: String
  val deviceType: EnumDeviceType
  val password: String
  val rememberMe: Boolean
}